Installation/Source/requestInbox.source.json

```diff
@@ -0,0 +1,35 @@
+{
+  "title": "VrijBRP Request Inbox",
+  "description": "VrijBRP Request Inbox",
+  "$id": "https://vrijbrp.nl/sources/vrijbrp.requestInbox.source.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Gateway.schema.json",
+  "version": "0.0.1",
+  "location": "https://request-inbox.vrijbrp.nl",
+  "isEnabled": true,
+  "type": "json",
+  "authorizationHeader": "Authorization",
+  "auth": "oauth",
+  "authenticationConfig": {
+    "tokenUrl": "https://request-inbox.vrijbrp.nl/auth",
+    "tokenField": "token"
+  },
+  "authorizationPassthroughMethod": "base_auth",
+  "username": "!ChangeMe!",
+  "password": "!ChangeMe!",
+  "documentation": null,
+  "loggingConfig": {
+    "callMethod": true,
+    "callUrl": true,
+    "callQuery": true,
+    "callContentType": true,
+    "callBody": true,
+    "responseStatusCode": true,
+    "responseContentType": true,
+    "responseBody": true,
+    "maxCharCountBody": 500,
+    "maxCharCountErrorBody": 2000
+  },
+  "configuration": {
+    "verify": true
```

Comment:
These are default values, you should not need to set these

Reply:
I like to have it in case I need to add more config